hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1596,14 +1596,15 @@
 
 <property>
   <name>fs.s3a.connection.establish.timeout</name>
-  <value>5000</value>
-  <description>Socket connection setup timeout in milliseconds.</description>
+  <value>5s</value>
+  <description>Socket connection setup timeout in milliseconds; this will be retried
+    more than once.</description>
 </property>
 
 <property>
   <name>fs.s3a.connection.timeout</name>
-  <value>200000</value>
-  <description>Socket connection timeout in milliseconds.</description>
+  <value>200s</value>
+  <description>Socket connection timeout.</description>
```

Comment:
For both of them, is it valuable to mention in the description that milliseconds is the default unit unless user specifies any other unit explicitly?

Reply:
actually put it into performance.md to avoid merge conflict, cut out old stuff from index.md and point at new section